scripts/geo_to_xml.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/env bash
+
```

Comment:
A brief comment here for what this script does and a copy paste of what echo_usage prints would be useful.

Reply:
Done